fvm/src/kernel/default.rs

```diff
@@ -738,15 +738,16 @@ where
 
     // TODO(M2) merge new_actor_address and create_actor into a single syscall.
     fn create_actor(&mut self, code_id: Cid, actor_id: ActorID) -> Result<()> {
-        let typ = self
+        let singleton = self
             .get_builtin_actor_type(&code_id)
-            .ok_or_else(|| syscall_error!(Forbidden; "can only create built-in actors"))?;
-
-        if typ.is_singleton_actor() {
+            .as_ref()
+            .map(Type::is_singleton_actor)
+            .unwrap_or(false);
+        if singleton {
```

Comment:
https://github.com/filecoin-project/builtin-actors/issues/492

Reply:
added TODO for this, pointing to the issue.